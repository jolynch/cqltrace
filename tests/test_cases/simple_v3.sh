#!/bin/bash
./cqltrace -b $1
