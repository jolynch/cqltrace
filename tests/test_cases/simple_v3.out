127.0.0.1:51566|SELECT * FROM system.peers|ONE|BINDS=NA|0.001220000
127.0.0.1:51566|SELECT * FROM system.local WHERE key='local'|ONE|BINDS=NA|0.001530000
127.0.0.1:51566|SELECT * FROM system.schema_keyspaces|ONE|BINDS=NA|0.001194000
127.0.0.1:51566|SELECT * FROM system.schema_columnfamilies|ONE|BINDS=NA|0.004687000
127.0.0.1:51566|SELECT * FROM system.schema_columnfamilies|ONE|BINDS=NA|0.007758000
127.0.0.1:51566|SELECT * FROM system.schema_columnfamilies|ONE|BINDS=NA|0.007775000
127.0.0.1:46780|USE "test"|ONE|BINDS=NA|0.001862000
127.0.0.1:59028|USE "test"|ONE|BINDS=NA|0.002464000
127.0.0.1:51572|USE "test"|ONE|BINDS=NA|0.002932000
127.0.0.1:51572|0B790BCC6FE3CA56322F602C0FF88D2D|LOCAL_ONE|BINDS=30:35:31:36|0.022321000
127.0.0.1:51572|654772B3D4CFA251A111B33F9F6F9338|LOCAL_ONE|BINDS=30:35|0.001244000
127.0.0.1:51572|654772B3D4CFA251A111B33F9F6F9338|LOCAL_ONE|BINDS=31:36|0.001012000
127.0.0.1:51572|654772B3D4CFA251A111B33F9F6F9338|LOCAL_ONE|BINDS=32:37|0.001109000
127.0.0.1:46780|654772B3D4CFA251A111B33F9F6F9338|LOCAL_ONE|BINDS=33:38|0.008738000
127.0.0.1:46780|654772B3D4CFA251A111B33F9F6F9338|LOCAL_ONE|BINDS=34:39|0.001903000
127.0.0.1:46780|654772B3D4CFA251A111B33F9F6F9338|LOCAL_ONE|BINDS=35:3130|0.001302000
127.0.0.1:59028|654772B3D4CFA251A111B33F9F6F9338|LOCAL_ONE|BINDS=36:3131|0.008241000
127.0.0.1:46780|654772B3D4CFA251A111B33F9F6F9338|LOCAL_ONE|BINDS=37:3132|0.003386000
127.0.0.1:51572|654772B3D4CFA251A111B33F9F6F9338|LOCAL_ONE|BINDS=38:3133|0.000945000
127.0.0.1:46780|654772B3D4CFA251A111B33F9F6F9338|LOCAL_ONE|BINDS=39:3134|0.000818000
